app/views/shared/_stats.html.erb

```diff
@@ -0,0 +1,15 @@
+<% @user ||= current_user %>
+<div class="stats">
+  <a href="<%= following_user_path(@user) %>">
+    <strong id="following" class="stat">
+      <%= @user.following.count %>
```

Comment:
nó không lấy các following từ CSDL.nó thực hiện tính toán trong CSDL.yêu cầu CSDL đếm các following với user_id
gọi chiều dài trên mảng bằng hàm count()